lms/djangoapps/instructor/utils.py

```diff
@@ -278,7 +292,7 @@ def ora2_data_queries():
 WHERE `student`.`item_type`="openassessment" AND `student`.`course_id`=%s
     """
 
-    return RAW_QUERY
+    return RAW_QUERY_1 + RAW_QUERY_3 + RAW_QUERY_2
```

Comment:
If it proves impossible to restructure the query above into a single parameterized string, we should probably still reorder/rename these variables to correspond to their order in the query.  I'd like this code to be crystal clear to people, so we don't have to worry about SQL attacks/misuse.